docs/index.html

```diff
@@ -388,9 +409,12 @@ <h2>#TwitterThrowback</h2>
                     if (tweets[year].length) {
                         hasContent = true;
                         tweetsHtml.push(`<div>`);
-                        tweetsHtml.push(`<h3>${year}</h3>`);
+                        tweetsHtml.push(`<div class="year_button" onclick="minimizer(event)">${year}</div>`);
```

Comment:
Forgot to update this?

```suggestion
                        tweetsHtml.push(`<div class="year_button" onclick="showOrHideTweetsInYear(event)">${year}</div>`);

```